cmd/bench/cmd/root.go

```diff
@@ -0,0 +1,39 @@
+// Copyright Contributors to the Mir project
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	id      string
+	nrNodes int
+	verbose bool
+
+	rootCmd = &cobra.Command{
+		Use:   "bench",
+		Short: "Mir benchmarking tool",
```

Comment:
Now it looks more like "ISS benchmarking tool".
Please keep in mind that Mir != ISS.

Reply:
It seems that so far Mir's main protocol implementation is ISS. We can generalize it later.

Reply:
Please see the discussion section [here](https://github.com/protocol/ConsensusLab/issues/9#issuecomment-1208245086)

Reply:
Indeed, this tool currently supports benchmarking ISS instantiations only, but it could be extended in future, e.g. we could add CLI options or some kind of flexible configuration.

Reply:
It looks like another layer of abstraction / encapsulation could be useful here that would group related modules together to something we could call "system" or "application" or something like that. I actually created an issue for that yesterday (#215).

We could then create separate benchmarks specific to those module compositions, potentially (re-)using some general benchmarking package (e.g. a customizable interceptor with a configurable event selector function and event processor function etc.). Benchmarking "pure Mir" is anyway only meaningful to obtain some very low-level microbenchmarks (like events processed per second or so). Not saying that it wouldn't be useful, but the most practically meaningful benchmarks inherently need to be model- (or model-composition-) specific.

So for now, we could 1) choose a name for the TOB / SMR implementation we are building now (which is the first and so far only actual application of Mir) 2) rename this benchmark accordingly. WDYT?